.github/workflows/ci-standard-validator.yml

```diff
@@ -0,0 +1,25 @@
+name: ci-standard-validator
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'packages/standard-validator/**'
+  pull_request:
+    branches: ['*']
+    paths:
+      - 'packages/standard-validator/**'
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/standard-validator
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
```

Comment:
Please replace `18.x` with `20.x`? Some other CI uses `18.x`, but currently `20.x` is better.